File: C/Assignments/Assignment 6/Assignment 1 with Function/q5.c
// Write a program to check whether a person is eligible to vote (age = 18).

void check() {
	// take input
	int age ;
	printf("enter age : ");
	scanf("%d" , &age);
	
	// check and print person is eligible to vote 
	if(age >= 18)
		printf("eligible for vote");
		
	else
		printf("not eligible for vote");
}


void main() {
	check();
	
}

File: C to Java/Assignments/Assignment 1/q5.java
// Write a program to check whether a person is eligible to vote (age = 18).

import java.util.Scanner;

public class q5 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        // take input
	int age ;
	System.out.printf("enter age : ");
	age = sc.nextInt();
	
	// check and print person is eligible to vote 
	if(age >= 18)
		System.out.printf("eligible for vote");
		
	else
		System.out.printf("not eligible for vote");

        }
}